plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/AbstractKuduConnectorTest.java

```diff
@@ -482,57 +507,118 @@ protected String errorMessageForInsertNegativeDate(String date)
     }
 
     @Override
+    @Test(timeOut = 60_000, invocationCount = 4)
     public void testInsertRowConcurrently()
+            throws Exception
     {
-        // TODO Support these test once kudu connector can create tables with default partitions
-        throw new SkipException("TODO");
+        // TODO Remove this overriding test once kudu connector can create tables with default partitions
+        int threads = 4;
+        CyclicBarrier barrier = new CyclicBarrier(threads);
+        ExecutorService executor = newFixedThreadPool(threads);
+        try (TestTable table = new TestTable(getQueryRunner()::execute, "test_insert",
```

Comment:
You can extract a method to create a table likes `createTableWithDefaultColumns` in base class. Subsequent tests are same.

Reply:
@ebyhr Thanks! Please see my update. Not sure if it's exactly what you meant, please correct me if it's not :)